extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -167,8 +177,9 @@ export class AmpA4A extends AMP.BaseElement {
     // from cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
     /** @private {?ArrayBuffer} */
     this.creativeBody_ = null;
-    /** @private {?string} */
-    this.experimentalNonAmpCreativeRenderMethod_ = null;
+    /** @private {!CROSS_ORIGIN_RENDERING_MODE} */
+    this.experimentalNonAmpCreativeRenderMethod_ =
```

Comment:
Given this now includes the default (currently cache, but likely soon to be safe/name frame for ios), let's remove "experimental" from the name